lib/spack/docs/module_file_support.rst

```diff
@@ -0,0 +1,659 @@
+===============================
+Integration with module systems
+===============================
+
+The use of module systems to manage user environment in a controlled way
+is a common practice at HPC centers that is often embraced also by individual
+programmers on their development machines. To support this common practice
```

Comment:
@scheibelp I general for any docs PR of mine : I find it useful! As a non-native speaker it improves my English and makes docs better :smile: